tests/unit/containers/my-quizzes/CreateOrEditQuizContainer.spec.jsx

```diff
@@ -109,44 +114,51 @@ describe('CreateOrEditQuizContainer without id', () => {
 })
 
 describe('CreateOrEditQuizContainer with id', () => {
-  beforeAll(() => {
+  beforeEach(() => {
     useParams.mockReturnValue({ id: mockId })
-    mockAxiosClient
-      .onGet(new RegExp(URLs.quizzes.get.replace(':id', mockId)))
-      .reply(200, {
+
+    getQuizSpy = vi.spyOn(ResourceService, 'getQuizQuery').mockResolvedValue({
+      data: {
         _id: mockId,
         title: 'Mock title',
         description: 'Mock description',
-        category,
+        category: '665799d795ab9dbdd7ad40df',
+        settings: {
+          view: 'Stepper',
+          shuffle: false,
+          pointValues: true,
+          scoredResponses: true,
+          correctAnswers: true
+        },
         items: []
-      })
+      }
+    })
   })
 
   afterEach(() => {
     vi.clearAllMocks()
   })
 
-  it('should save quiz with category and questions', async () => {
-    await waitFor(() => renderComponent({ category, questions: [] }))
-
-    const saveBtn = screen.getByText('common.save')
-    fireEvent.click(saveBtn)
+  it('should call ResourceService.getQuizQuery with the correct id when loading the quiz', async () => {
+    await waitFor(() => {
+      renderComponent()
+    })
```

Comment:
`waitFor` here is not needed
```suggestion
      renderComponent()
```